applications/oci-transaction-lifecycle.md

```diff
@@ -83,10 +83,8 @@ Multiple members of the OCI team have worked on the EOSIO code base.  Kevin Heif
 
 ## Development Status
 
-If you've already started implementing your project or it is part of a larger repository, please provide a link and a description of the code here. In any case, please provide some documentation on the research and other work you have conducted before applying. This could be:
-
 - [EOS API Blue Paper](https://medium.com/eos-network-foundation/api-blue-paper-e78c0be0d878)
-- references to conversations you might have had related to this project with anyone from the EOS Network Foundation (Yves La Rose discussed).
+- conversations related to this project with stakeholders from the EOS Network Foundation (Yves La Rose discussed).
```

Comment:
This looked like it was a note that needed to be expanded upon before submission